src/test/java/org/folio/processing/configuration/ProfileManagerTest.java

```diff
@@ -0,0 +1,48 @@
+package org.folio.processing.configuration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+
+class ProfileManagerTest {
```

Comment:
Sonar analysis result: [20.2% Coverage on New Code] (required ≥ 80%)

Reply:
It's because tests in pipeline doesn't run for "asa-extended" profile, so the new code which is under the profile is not covered. We highlighted the problem to devops but still don't have an answer.

Reply:
We're going to get rid of profile approach, so the implementation will be updated